src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -19,70 +20,75 @@ public String getTestedClassName() {
-                {"print_reads.sam", ".sam"},
-                {"print_reads.sam", ".bam"},
-                {"print_reads.bam", ".sam"},
-                {"print_reads.bam", ".bam"},
-                {"print_reads.sorted.queryname.sam", ".sam"},
-                {"print_reads.sorted.queryname.sam", ".bam"},
-                {"print_reads.sorted.queryname.bam", ".sam"},
-                {"print_reads.sorted.queryname.bam", ".bam"},
+                {"print_reads.sam", ".sam", null},
+                {"print_reads.sam", ".bam", null},
+                {"print_reads.sam", ".cram", "print_reads.fasta"},
+                {"print_reads.bam", ".sam", null},
+                {"print_reads.bam", ".bam", null},
+                {"print_reads.bam", ".cram", "print_reads.fasta"},
+                {"print_reads.cram", ".sam", "print_reads.fasta"},
+                {"print_reads.cram", ".bam", "print_reads.fasta"},
+                {"print_reads.cram", ".cram", "print_reads.fasta"},
 
-        };
-    }
-
-    @DataProvider(name="testingDataCRAM")
-    public Object[][] testingDataCRAM() {
-        return new String[][]{
+                {"print_reads.sorted.sam", ".sam", null},
+                {"print_reads.sorted.sam", ".bam", null},
+                {"print_reads.sorted.sam", ".cram", "print_reads.fasta"},
+                {"print_reads.sorted.bam", ".sam", null},
+                {"print_reads.sorted.bam", ".bam", null},
+                {"print_reads.sorted.bam", ".cram", "print_reads.fasta"},
                 {"print_reads.sorted.cram", ".sam", "print_reads.fasta"},
                 {"print_reads.sorted.cram", ".bam", "print_reads.fasta"},
                 {"print_reads.sorted.cram", ".cram", "print_reads.fasta"},
-                {"print_reads.sorted.sam", ".cram", "print_reads.fasta"},
-                {"print_reads.sorted.bam", ".cram", "print_reads.fasta"},
+
+                {"print_reads.sorted.queryname.sam", ".sam", null},
+                {"print_reads.sorted.queryname.sam", ".bam", null},
+                {"print_reads.sorted.queryname.sam", ".cram", "print_reads.fasta"},
+                {"print_reads.sorted.queryname.bam", ".sam", null},
+                {"print_reads.sorted.queryname.bam", ".bam", null},
+                {"print_reads.sorted.queryname.bam", ".cram", "print_reads.fasta"},
                 {"print_reads.sorted.queryname.cram", ".sam", "print_reads.fasta"},
                 {"print_reads.sorted.queryname.cram", ".bam", "print_reads.fasta"},
-
-                //comparing queryname sorted cram files blows up: https://github.com/broadinstitute/gatk/issues/1271
-//                {"print_reads.sorted.queryname.cram", ".cram", "print_reads.fasta"},
-//                {"print_reads.sorted.queryname.sam", ".cram", "print_reads.fasta"},
-//                {"print_reads.sorted.queryname.bam", ".cram", "print_reads.fasta"}
+                {"print_reads.sorted.queryname.cram", ".cram", "print_reads.fasta"}
```

Comment:
If not, might want to add a name-sorted cram produced by a recent htsjdk as a test case, since internally the format is very different than it was before.